### ISSUE TITLE
resource not found problem while testing

### ISSUE THREAD

**Issue by sachin1998-vyas** [jmgtan/stock_dashboard#2]
when i test everytime it says secret manager doesn't find specified key

**Comment by jmgtan** [comment]
Which part of the guide are you in?

**Comment by sachin1998-vyas** [comment]
first its problem with creating secret manage keys and thn at last when i test my intraday function it says secret manager doesn't find specified keys

**Comment by markramrattan** [comment]
I am having the same problem with secrets manager not finding the specified key. Can't get this to work with the instructions on readme.

**Comment by jmgtan** [comment]
I retested it again and found a small discrepancy in the example config-exports.js. If you look at the CLI that creates the secret, it uses the format `stockMonitoring/<env>/backend`, but in the config-exports.js it is `stockMonitoringDashboard/<env>/backend`. Just make sure that the config-exports.js matches the secret name that you used when you created the secret. After correcting that the Lambda function is able to pick up the secret.

**Comment by markramrattan** [comment]
I did notice the change from stockMonitoringDashboard / stockMonitoring and made sure they matched. That didn't fix it for me over the weekend. On your screenshot picture, of lambda function mine wasn't showing connected to AWS SecretsManager / Amazon CloudWatch Logs / Amazon S3. Let me try again now. Thanks for updating also <value of user pool id> on the read me file :)  I knew you didn't mean eu-west-1 for that part.

**Comment by sachin1998-vyas** [comment]
i already changed it and make sure they match but still its not working please test it back and help me

**Comment by jmgtan** [comment]
@sachin1998-vyas can you share a screenshot of the error and your config-exports.js? I retested with a fresh setup earlier and couldn't replicate the error.

**Comment by sachin1998-vyas** [comment]
![error](https://user-images.githubusercontent.com/51750064/86165039-607f4800-bb30-11ea-84a2-b170d5e818be.PNG)
![error@](https://user-images.githubusercontent.com/51750064/86165046-61b07500-bb30-11ea-89be-d0bca5e1ca6d.PNG)

**Comment by sachin1998-vyas** [comment]
i have shared both config.js and error screenshot

**Comment by jmgtan** [comment]
@sachin1998-vyas am retesting now on us-east-1 just to make sure, can you confirm the secrets are in secrets manager via the console?

**Comment by jmgtan** [comment]
@sachin1998-vyas @markramrattan nevermind, I found the problem. Will upload a fix in a few mins.

**Comment by sachin1998-vyas** [comment]
![secreat](https://user-images.githubusercontent.com/51750064/86166282-29aa3180-bb32-11ea-8e76-6678a5cce11f.PNG)

**Comment by sachin1998-vyas** [comment]
![key command](https://user-images.githubusercontent.com/51750064/86166305-329b0300-bb32-11ea-949f-ec5c0251e2db.PNG)

**Comment by sachin1998-vyas** [comment]
last image key command was not valid so i changed it like this to create secrets
aws secretsmanager create-secret --name "stockMonitoring/local/backend" --secret-string '[{"Key":"username","Value":"samy@gmail.com"},{"Key":"password","Value":"srv12345"}]'

**Comment by jmgtan** [comment]
@sachin1998-vyas am just testing the fix now, will update once I have it committed.

**Comment by sachin1998-vyas** [comment]
ok sure, thanks to review this error for us i am waiting to see it work

**Comment by jmgtan** [comment]
@sachin1998-vyas @markramrattan Updated the GetIntradayPrices/index.js, just pull and then redeploy the lambda function. I noticed that the AlphaVantage feed doesn't provide intraday prices for some of the stock symbols like APPL, so I added an error handling to account for it.

**Comment by sachin1998-vyas** [comment]
can you look at create secret command too to make sure its correct

**Comment by jmgtan** [comment]
@sachin1998-vyas use powershell, not CMD

**Comment by sachin1998-vyas** [comment]
![why](https://user-images.githubusercontent.com/51750064/86182174-16588f80-bb4d-11ea-99d5-76f421b67a3d.PNG)

**Comment by sachin1998-vyas** [comment]
this time everything run successfully still getting error

**Comment by jmgtan** [comment]
@sachin1998-vyas in the `config-exports.js` file, did you update the `appsync_endpoint_url` parameter? Since you're deploying via Amplify console, you can get the endpoint URL by going to AppSync, selecting the app, and click settings. You should be able to find the API URL from that page.

I just tested it now in the us-east-1 region and using Amplify console and I'm able to stream the data in when the function is running.

**Comment by markramrattan** [comment]
Finally got it working, lost count of how many times i've tried to get it working :) I'll write up a new readme for instructions. The mix of launching local and deploying is confusing (Just local step by step would of worked better & another one for deploying - separate). Before setting out on doing this project, I would recommend doing amplify configure and creating a new IAM profile in the region you are wishing to work (eu-west-1). I was having resources being built in different regions. 

<img width="1431" alt="Screenshot 2020-07-04 at 14 00 17" src="https://user-images.githubusercontent.com/1474558/86513034-e0dacd00-bdfe-11ea-947c-ee52e8f6853e.png">

**Comment by jmgtan** [comment]
@markramrattan that's great to hear and thanks for the feedback. I'll make some changes to the README next week to separate it out. I am using a profile when I built and tested the code. I used ~/.aws/credentials to store the credentials am using and ~/.aws/config to set the region for that profile.

**Comment by jmgtan** [comment]
Closing as issue has been fixed.

**Comment by masdmh** [comment]
Is this still up to date?   Is this still the best way to go about producing such an app?
